### PR TITLE
update example code to use latest api functions

### PR DESCRIPTION
**Pull request by AbelTesfaye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ soccer_et.getTeamDetailJSON('ኢትዮጵያ ቡና').then(detail => console.log
 - [x] Team details
 - [ ] Player details
 - [ ] Top players list
-- [ ] News
```
Comment (brookmg) [inline README.md:50]: There was no change here?

**Comment by brookmg** [comment]
Good 👍

File: README.md (hunk 1 above)
Comment (AbelTesfaye) [reply README.md:50]: It was probably an accidental white-space change

Comment (brookmg) [reply README.md:50]: Makes sense

**Pull request merged**